src/views/UserDetail/UserDetail.js

```diff
@@ -309,6 +338,53 @@ class UserDetail extends React.Component {
     );
   }
 
+  showOpenTransactionsModal(json) {
+    this.setState({
+      showOpenTransactionModal: true,
+      openTransactions: json,
+    });
+  }
+
+  showDeleteUserModal(json) {
+    this.setState({
+      showDeleteUserModal: true,
+      openTransactions: json,
+    });
+  }
+
+  doClose = () => {
```

Comment:
Please use a proper name specifying what is going to be closed. Currently, it may suggest that the UserDetail view is closed.